auditbeat/auditbeat.reference.yml

```diff
@@ -1177,20 +1177,22 @@ logging.files:
 #logging.json: false
 
 
-#============================== Xpack Monitoring =====================================
-# auditbeat can export internal metrics to a central Elasticsearch monitoring cluster.
-# This requires xpack monitoring to be enabled in Elasticsearch.
-# The reporting is disabled by default.
+#============================== Xpack Monitoring ===============================
+# auditbeat can export internal metrics to a central Elasticsearch monitoring
+# cluster.  This requires xpack monitoring to be enabled in Elasticsearch.  The
+# reporting is disabled by default.
 
 # Set to true to enable the monitoring reporter.
-#xpack.monitoring.enabled: false
+#monitoring.enabled: false
 
 # Uncomment to send the metrics to Elasticsearch. Most settings from the
-# Elasticsearch output are accepted here as well. Any setting that is not set is
-# automatically inherited from the Elasticsearch output configuration, so if you
-# have the Elasticsearch output configured, you can simply uncomment the
-# following line, and leave the rest commented out.
-#xpack.monitoring.elasticsearch:
+# Elasticsearch output are accepted here as well.
+# Note that the settings should point to your Elasticsearch *monitoring* cluster.
+# Any setting that is not set is automatically inherited from the Elasticsearch
+# output configuration, so if you have the Elasticsearch output configured such
+# that it is pointing to your Elasticsearch monitoring cluster, you can simply
```

Comment:
Slightly off-topic but I wonder if the concept of a monitoring cluster would be more clear if we started referring to them as _dedicated_ monitoring clusters to indicate that monitoring is their sole purpose.

Reply:
I think in this case just "monitoring cluster" is appropriate. It could be the same ES cluster where the rest of the data is being stored as well OR it could be a dedicated monitoring cluster.

I agree that there are other places in our docs, etc. where we explicitly want to indicate a dedicated monitoring cluster and we should revise those places. Out-of-scope for this particular PR, obviously.